pkg/k8s/client.go

```diff
@@ -43,6 +47,13 @@ func (c *Client) GetPodDisruptionBudgets(namespace string) (*v1beta1.PodDisrupti
 	return c.clientSet.PolicyV1beta1().PodDisruptionBudgets(namespace).List(metav1.ListOptions{})
 }
 
+func (c *Client) prepareToApply(pod *apiv1.Pod) {
+	pod.ObjectMeta.SetResourceVersion("")
```

Comment:
ideally find this in kubernetes client somewhere ... they have it because they are doing it with `--export` 🤷‍♂ ... but see comment above